src/authenticated.js

```diff
@@ -1,17 +1,19 @@
 (function() {
     const onlyLoggedOut = document.querySelector('.only-loggedout'),
         onlyLogged = document.querySelector('.only-logged'),
-        onlyLoggedProfile = document.querySelector('.only-logged-profile');
+        onlyLoggedProfile = document.querySelector('.only-logged-profile'),
+        authReq = new XMLHttpRequest();
 
     // See if user is authenticated
-    fetch('/authenticated')
-    .then((resp) => resp.json())
-    .then((data) => {
+    authReq.open('GET', '/authenticated', true);
```

Comment:
Why did you remove fetch here? It's already supported by all major browsers and when 3.0.0 comes out it will also be supported by safari